docs/support/FAQ.md

```diff
@@ -0,0 +1,148 @@
+
+**semantic-release** takes care of updating the `package.json`’s version before publishing to [npm](https://www.npmjs.com).
+
+By default, only the published package will contains the version, which is the only place where it is *really* required, but the updated `package.json` will not be pushed to the Git repository
+
+However, the [`@semantic-release/git`](https://github.com/semantic-release/git) plugin can be used to push the updated `package.json` as well as other files to the Git repository.
+
+## How can I use a npm build script that requires the `package.json`’s version ?
+
+The `package.json`’s version will be updated by the `semantic-release` command just before publishing to [npm](https://www.npmjs.com), therefore it won't be available for scripts ran before the `semantic-release` command.
+
+As semantic-release uses the [npm CLI](https://docs.npmjs.com/cli/npm) to publish, all [npm hook scripts](https://docs.npmjs.com/misc/scripts#hook-scripts) will be executed. Therefore you can run your build script in the [`prepublishOnly`](https://docs.npmjs.com/misc/scripts#prepublish-and-prepare) hook. It will be executed after the `semantic-release` command update the `package.json`’s version and before publishing.
+
+## Is there a way to preview which version would currently get published?
+
+Yes with the [dry-run options](../usage/configuration.md#dryrun) which prints to the console the next version to be published and the release notes.
+
+## Can I use semantic-release with Yarn?
+
+If you are using a [local](../usage/installation.md#local-installation) **semantic-release** installation and run multiple CI jobs with different versions, the `yarn install` command will fail with Node < 8 as **semantic-release** require [Node >= 8](#why-does-semantic-release-require-node-version--8). See [yarnpkg/rfcs#69](https://github.com/yarnpkg/rfcs/pull/69).
+
+In order to run **semantic-release** with [Yarn](https://yarnpkg.com) you would need to:
+- Use a [global](../usage/installation.md#global-installation) **semantic-release** installation
+- Make sure to install and run the `semantic-release` command only in a CI jobs running with [Node >= 8](#why-does-semantic-release-require-node-version--8)
+
+If your CI environment provides [nvm](https://github.com/creationix/nvm) you can switch to Node 8 before installing and running the `semantic-release` command:
+
+```bash
+$ nvm install 8 && yarn install -g semantic-release && semantic-release
+```
+
+See the [CI configuration recipes](../recipes/README.md#ci-configurations) for more details on specific CI environments.
+
+## Can I use semantic-release to publish non-JavaScript packages?
+
+Yes, **semantic-release** is a Node CLI application but it can be used to publish any type of packages.
+
+To publish a non-JavaScript package you would need to:
+- Use a [global](../usage/installation.md#global-installation) **semantic-release** installation
```

Comment:
I think it makes sense to mention that you can just install it globally in CI if you don't want a package.json, but stating the user _needs_ to is misinformation.

Reply:
By "non-JavaScript package" I implied a project without a `package.json`. I added some clarification in #594